include/highfive/bits/H5Attribute_misc.hpp

```diff
@@ -86,6 +86,11 @@ inline void Attribute::read(T& array) const {
     converter.process_result(array);
 }
 
+File Attribute::getFile(){
+    hid_t file_id = H5Iget_file_id(_hid);
+    return File(file_id);
+}
```

Comment:
This is repeated 3 times, with pretty much identical code. Is there a way to unify this apart from what you have outlined in the PR description?

Reply:
Now I moved `getFile()` to a protected `Object`'s method and any derived uses `Object::getFile()`